### PR TITLE
more platform 6 fixes

### PR DESCRIPTION
**Pull request by mlangsdorf**
koji_import gets rendered properly
platforms is an optional argument in arrangement6 and create_orchestrate_build shouldn't raise an error if it is omitted.

### CODE REVIEW

File: osbs/build/plugins_configuration.py [hunk 1]
```diff
@@ -363,6 +363,15 @@ def render_inject_parent_image(self):
 
         self.pt.set_plugin_arg(phase, plugin, 'koji_parent_build', koji_parent_build)
 
+    def render_koji_import(self, use_auth=None):
+        if not self.pt.has_plugin_conf('exit_plugins', 'koji_import'):
+            return
+
+        koji_target = self.user_params.koji_target.value
+        if not self.pt.set_plugin_arg_valid('exit_plugins', 'koji_import', 'target', koji_target):
```
Comment (lcarva) [inline osbs/build/plugins_configuration.py]: Looking at `koji_import` plugin, I don't think `target` parameter is used at all.

File: tests/test_api.py [hunk 2]
```diff
@@ -530,8 +530,8 @@ def test_create_worker_build_ioerror(self, osbs):  # noqa
         )
     )
     @pytest.mark.parametrize(('platforms', 'raises_exception'), (
```
Comment (lcarva) [inline tests/test_api.py]: Any reason to keep `raises_exception` parameter if it's always `False`?

File: osbs/build/plugins_configuration.py (hunk 1 above)
Comment (mlangsdorf) [reply osbs/build/plugins_configuration.py]: Fixed, or at least, I'm not doing anything to render koji_import anymore.

File: tests/test_api.py (hunk 2 above)
Comment (mlangsdorf) [reply tests/test_api.py]: Fixed.

**Comment by lcarva** [comment]
Not updating release notes draft because this is bugfix to an unreleased feature.

**Pull request merged**